internal/view/sbommonitor/render.go

```diff
@@ -0,0 +1,31 @@
+package sbommonitor
+
+import (
+	"io"
+
+	"github.com/snyk/cli-extension-sbom/internal/snykclient"
+)
+
+func RenderMonitor(dst io.Writer, monitors []*snykclient.MonitorDependenciesResponse) (int, error) {
+	var mp = make([]monitorProjectComponent, len(monitors))
+
+	for i, m := range monitors {
+		c, err := generateMonitorProjectComponent(m.ProjectName, m.URI)
+		if err != nil {
+			return 0, err
+		}
+		mp[i] = c
+	}
+
+	monitor, err := generateMonitorComponent(mp)
+	if err != nil {
+		return 0, err
+	}
+
+	n, err := io.WriteString(dst, monitor.String())
+	if err != nil {
+		return n, err
+	}
+
+	return n, nil
```

Comment:
nit: could just be this?
```suggestion
	return io.WriteString(dst, monitor.String())
```